web_ui/server.py

```
@@ -2368,12 +2620,11 @@ def twofa_disable():
         return jsonify({"error": "2FA not enabled", "success": False}), 400
 
-    with contextlib.suppress(Exception):
-        # Remove TOTP secret and recovery hashes from the encrypted user record
-        with suppress(Exception):
-            auth_mod.set_totp_secret(None)
-        with suppress(Exception):
-            auth_mod.set_recovery_hashes([])
+    try:
+        auth_mod.set_totp_secret(None)
+        auth_mod.set_recovery_hashes([])
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)